Modules/_winapi.c

```diff
@@ -1553,6 +1553,28 @@ _winapi_ReadFile_impl(PyObject *module, HANDLE handle, DWORD size,
     return Py_BuildValue("NI", buf, err);
 }
 
+/*[clinic input]
+_winapi.ResumeThread
+
+    handle: HANDLE
+    /
+[clinic start generated code]*/
+
+static PyObject *
+_winapi_ResumeThread_impl(PyObject *module, HANDLE handle)
+/*[clinic end generated code: output=eadd6124a82b305b input=8800116464bd88c2]*/
+{
+    DWORD result;
+    Py_BEGIN_ALLOW_THREADS
+    result = ResumeThread(handle);
+    Py_END_ALLOW_THREADS
+
+    if (result == -1)
+        return PyErr_SetFromWindowsErr(GetLastError());
+
+    return PyLong_FromLong((int) result);
```

Comment:
There's already a converter for the `DWORD` type, which is used for `GetExitCodeProcess`, `GetLastError`, and `GetFileType`. Just change the clinic input to return a `DWORD`.